skimage/measure/mc_meta/visual_test.py

```diff
@@ -3,70 +3,118 @@
 data.
 """
 
+from __future__ import print_function

```

Comment:
We don't support Py2.7 on master so this is unnecessary.

Reply:
I added it for backport to 0.14.x

Reply:
My policy generally has been to leave any backport-required code for the backport, but to each their own. ;)